packages/contracts-bedrock/contracts/test/Semver.t.sol

```diff
@@ -19,7 +19,8 @@ contract Semver_Test is CommonTest {
     /**
      * @notice Deploy a Semver contract
      */
-    function setUp() external {
+    function setUp() public virtual override {
+        super.setUp();
```

Comment:
Seems we don't _need_ this, is it going to slow things down at all?

Reply:
There are definitely some that we don't need, happy to follow up with additional cleanup in the future